src/components/ProfileInformation.jsx

```diff
@@ -48,104 +50,106 @@ function ProfileInformation() {
-              onChange={(e) => onChange({ brTelefon: e })}
-              placeholder="Telefon"
-              value={form && form["brTelefon"]}
-            />
-          </div>
-          <div>
-            <Button
-              className="float-right"
-              onClick={() => onSubmit()}
-              label="Promeni"
-            />
-          </div>
-        </div>
-      </Card>
+    return (
+        <div className="flex flex-col">
+            {error && (
+                <Alert
+                    design="danger"
+                    text="Neuspesno menjanje informacije"
+                    onDismiss={() => setError(null)}
+                ></Alert>
+            )}
+            {success && (
+                <Alert
+                    design="success"
+                    text="Uspesno promenjeni podaci"
+                    onDismiss={() => setSuccess(null)}
+                ></Alert>
+            )}
+            <div className="flex items-center">
+                <div className="w-[200px]">Limit</div>
+                <div>
+                    {form && form["limit"]}
+                </div>
+            </div>
+            <Card title="Osnovno">
+                <div class="flex flex-col gap-5">
+                    <div className="flex justify-between items-center">
+                        <div className="w-[100px]">Ime</div>
+                        <TextField
+                            className="grow"
+                            onChange={(e) => onChange({ ime: e })}
+                            placeholder="Ime"
+                            value={form && form["ime"]}
+                        />
+                    </div>
+                    <div className="flex justify-between items-center">
+                        <div className="w-[100px]">Prezime</div>
+                        <TextField
+                            className="grow"
+                            onChange={(e) => onChange({ prezime: e })}
+                            placeholder="Prezime"
+                            value={form && form["prezime"]}
+                        />
+                    </div>
+                    <div className="flex justify-between items-center">
+                        <div className="w-[100px]">JMBG</div>
+                        <TextField
+                            className="grow"
+                            onChange={(e) => onChange({ jmbg: e })}
+                            placeholder="JMBG"
+                            value={form && form["jmbg"]}
+                        />
+                    </div>
+                    <div className="flex justify-between items-center">
+                        <div className="w-[100px]">Pozicija</div>
+                        <Select
+                            className="grow"
+                            onChange={(e) => onChange({ pozicija: e })}
+                            options={[BANK_POSITIONS.ADMIN, BANK_POSITIONS.ADMIN_GL]}

```

Comment:
Pre je bilo i BANK_POSITIONS.ROLE_AGENT i BANK_POSITIONS.ROLE_SUPERVISOR unutar options liste. Treba to da vratis.